src/app/header/Header.tsx

```diff
@@ -32,17 +40,21 @@ export function Header({ onSearchChange }: HeaderProps): JSX.Element {
           _hover={{ bg: 'transparent' }}
           _active={{ bg: 'transparent' }}
           _focus={{ border: 'none' }}
-          ml={5}
+          p="0"
         >
           <Image src={process.env.PUBLIC_URL + '/assets/logo.png'} alt="clockwork" h="40px" />
         </Button>
       </GridItem>
-      <GridItem colStart={2}>
+      <GridItem
+        colStart={{ base: 1, sm: 1, md: 2 }}
+        colSpan={{ base: 1, sm: 1, md: 1 }}
+        rowStart={{ base: 3, sm: 2, md: 1 }}
+      >
         <Center>
           <Search onChange={onSearchChange} />
         </Center>
       </GridItem>
-      <GridItem colStart={3} alignContent="flex-end">
+      <GridItem colStart={{ base: 1, sm: 2, md: 3 }} rowStart={{ base: 2, sm: 2, md: 1 }} alignContent="flex-end">
```

Comment:
is there a way you can move the term buttons closer to the search bar? Maybe move it into the same grid item and give it a margin on the left?
IMO it makes sense to be close to the search bar since they're directly related.